vcr/holder/wallet.go

```diff
@@ -118,12 +119,16 @@ func (h wallet) buildJWTPresentation(ctx context.Context, subjectDID did.DID, cr
 		jwt.IssuerKey:  subjectDID.String(),
 		jwt.SubjectKey: subjectDID.String(),
 		jwt.JwtIDKey:   id.String(),
+		"nonce":        crypto.GenerateNonce(),
```

Comment:
challenge is not the same as nonce? I'd say nonce is a technical random value included in the signature to prevent replay attacks, while challenge is a functional value (like the IRMA contract message)? In JSON-LD Proofs they're 2 different fields anyways.

Reply:
openid4vp requires it to be passed